controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/DeploymentIssueReporter.java

```diff
@@ -74,22 +74,29 @@ private void maintainDeploymentIssues(List<Application> applications) {
             else
                 controller().applications().setJiraIssueId(application.id(), Optional.empty());
 
-        // TODO: Do this when version.confidence is BROKEN instead?
+        // TODO: Do this when version.confidence is BROKEN instead? Or, exclude above those upgrading to BROKEN version?
```

Comment:
> Or, exclude above those upgrading to BROKEN version?

What does that even mean?

Reply:
The loop above picks out failing applications. We should leave out those upgrading to a BROKEN version. I guess a pair of commas could have helped (although it would still be awkward).

Reply:
Ok.